docs/reference/concepts/deployments.md

```diff
@@ -1,81 +1,51 @@
 
-## Configuring your deployment
-
-To ensure the SpatialOS Runtime starts [server-workers](https://docs.improbable.io/reference/latest/shared/concepts/workers-load-balancing#server-workers) correctly, you need to ensure the launch configuration file has the worker set up correctly. See the [Worker launch configuration](https://docs.improbable.io/reference/latest/shared/worker-configuration/launch-configuration#worker-launch-configuration) in the SpatialOS documentation for guidance on how to define the worker launch configurations for both server-workers and client-workers.
-
-## Local deployment
-
-To start a local deployment, either:
-
-* In your Unity Editor, from the menu select **SpatialOS** > **Local Launch**. (This runs the default launch configuration.)
-
-    or
-* Open a terminal window and from the root of your game project directory run `spatial local launch` via the [`spatial` CLI](https://docs.improbable.io/reference/latest/shared/glossary#the-spatial-command-line-tool-cli).
-
-(See the [SpatialOS documentation](https://docs.improbable.io/reference/latest/shared/spatial-cli/spatial-local-launch) for details of the launch configurations.)
+Once the deployment is running, you can connect clients to it in order to play your game.
 
-This starts a local version of the SpatialOS Runtime on your development machine together with all server-workers specified in the launch configuration you used.
-
-It’s done when you see the following message in the terminal: `SpatialOS ready. Access the Inspector at http://localhost:21000/inspector`.
+You can run a deployment on your development machine (a "local deployment") or in the cloud (a "cloud deployment").
 
-To start your client-workers, from a terminal window, in any directory, run `spatial local worker launch <YourExternalWorker> <YourLaunchConfig>`, replacing the `<example-content>` with names relevant to your game. Use the [Inspector](https://docs.improbable.io/reference/latest/shared/operate/inspector) to look at the current state of your game world.
-
-## Cloud deployment
+## Configuring your deployment
 
-Follow this section to start a cloud deployment.
+Before you launch a deployment, you need to ensure that it is configured correctly. There are two parts to configuring a deployment: the launch configuration and the worker configurations.
 
-### Setup the SpatialOS project name
+### Launch configuration
 
-You are allocated an empty SpatialOS project in the cloud when you sign up to SpatialOS; you use this to deploy your game, but to do this you need to tell the GDK the name of your allocated SpatialOS project so it knows where to deploy your game to.
+The launch configuration file specifies the parameters of your game world as well as your load balancing configuration. The load balancing configuration determines the worker instances that the SpatialOS Runtime will start.
```

Comment:
should we call out that the LB config also lets you define workers that the runtime should balance but not manage itself (e.g. our local launch configs)